reactioncommerce/reaction-next-starterkit
# External theme file to control all the component's style.

Impact: **major**
Type: **refactor**

## Issue
To change the style of a component, that component had to be extended and then replaced because the style for every component was inside that component. 
This is a lot of effort for just changing the style of a single component.

## Solution
This PR extracts the style of all the components to a single file that can be over-ridden by anyone who wants to change the theme of the app.

## Breaking changes
None

## Testing
1. Start the App, no styling should be broken.
1. Change some style in `src/lib/theme/reactionTheme.js`, the style change should be reflected in the UI.
